plugins/modules/field_info.py

```diff
@@ -1,7 +1,20 @@
 #!/usr/bin/python
 # -*- coding: utf-8 -*-
 #
-# (c) 2021, 1Password & Agilebits (@1Password)
+#  Copyright (C) 2021  1Password & Agilebits (@1Password)
+#
+#  This program is free software: you can redistribute it and/or modify
```

Comment:
this license is incorrect, we use MIT not GNU GPL

Reply:
This is a requirement from Ansible to do so. See [Collection licensing requirements](https://docs.ansible.com/ansible/latest/community/collection_contributors/collection_requirements.html#id31)

If setting MIT, the sanity check will fail with the following error:
```
ERROR: Found 1 validate-modules issue(s) which need to be resolved:
ERROR: plugins/modules/item_info.py:0:0: missing-gplv3-license: GPLv3 license header not found in the first 20 lines of the module
```

Not sure if we have the ability to skip it.

Reply:
Aha, I see :) In that case, can you please update `LICENSE.md` as well?

Reply:
updated